client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -29,16 +44,35 @@ class HostCredentialsPage extends Component {
 		this.setFieldValue( fieldName, e.target.value );
 	}
 
+	redirectToCart = () => {
+		upgradesActions.addItem( cartItems.guidedTransferItem() );
+		page.redirect( `/checkout/${ this.props.siteSlug }` );
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( nextProps.isAwaitingPurchase === true ) {
```

Comment:
What other truthy values are you expecting? We're usually not so explicit about these comparisons. Instead, this is just as effective:

```js
if ( nextProps.isAwaitingPurchase ) {
```